### PR TITLE
feat(sanity): Update presence menu button for inviting new collaborators

### PR DESCRIPTION
**Pull request by drewlyton**
### Description

This PR makes some updates to the presence dropdown menu action button to encourage seat expansion.

SCR-20240826-oisj

- Button text has been changed to "Invite collaborators"
- `@sanity/icons` has been updated to version 3.4.0
- Icon has been changed to new AddUserIcon
- Link has been updated to take the user directly to the project members tab

[Linear issues](https://linear.app/sanity/issue/GRO-2406/studio-update-manage-members-dropdown-menu-item)

### What to review

Review the text and link changes.

### Testing

Change does not affect or require new automated tests. To test the change, navigate to a studio document with someone else present and open the presence menu by clicking on the user avatars.

### Notes for release

No notes for release

### CODE REVIEW

**Review by ricokahler — DISMISSED** [review]

File: packages/sanity/src/core/studio/components/navbar/presence/PresenceMenu.tsx [hunk 1]
```diff
@@ -107,8 +107,8 @@ export function PresenceMenu() {
 
             <MenuItem
               as="a"
-              href={`https://sanity.io/manage/project/${projectId}`}
-              iconRight={CogIcon}
+              href={`https://sanity.io/manage/project/${projectId}/members?invite=true`}
+              iconRight={AddUserIcon}
```
Comment (joneidejohnsen) [inline packages/sanity/src/core/studio/components/navbar/presence/PresenceMenu.tsx:111]:
```suggestion
              iconLeft={AddUserIcon}
```

File: packages/sanity/src/core/i18n/bundles/studio.ts [hunk 2]
```diff
@@ -1100,7 +1100,7 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   /** Title of the alert for studio users when packages in their studio are out-of-date */
   'package-version.new-package-available.title': 'Sanity Studio is ready to update!',
   /** Label for action to manage members of the current studio project */
```
Comment (joneidejohnsen) [inline packages/sanity/src/core/i18n/bundles/studio.ts:1102]:
```suggestion
  /** Label for action to invite members to the current studio project */
```

File: packages/sanity/src/core/i18n/bundles/studio.ts [hunk 3]
```diff
@@ -1100,7 +1100,7 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
...
-  'presence.action.manage-members': 'Manage members',
+  'presence.action.manage-members': 'Invite collaborators',
```
Comment (joneidejohnsen) [inline packages/sanity/src/core/i18n/bundles/studio.ts:1103]: Suggest we call this "Invite members" so we are consistent with the user dropdown item.

Also wondering what we do with the i18n – do we need to update this in the language packages too?

**Review by joneidejohnsen — CHANGES_REQUESTED** [review]

**Comment by joneidejohnsen** [comment]
Suggest we also update the icon for "Invite members" in the user dropdown for consitency across the app.

image

File: packages/sanity/src/core/i18n/bundles/studio.ts (hunk 3 above)
Comment (drewlyton) [reply packages/sanity/src/core/i18n/bundles/studio.ts]: String is updated - no idea what the process is for inputing translations

File: packages/sanity/src/core/studio/components/navbar/navDrawer/ManageMenu.tsx [hunk 4]
```diff
@@ -39,8 +39,8 @@ export function ManageMenu() {
             <Button
               aria-label={t('user-menu.action.invite-members-aria-label')}
               as="a"
-              href={`https://sanity.io/manage/project/${projectId}/members`}
-              icon={UsersIcon}
+              href={`https://www.sanity.io/manage/project/${projectId}/members`}
```
Comment (joneidejohnsen) [inline packages/sanity/src/core/studio/components/navbar/navDrawer/ManageMenu.tsx:42]:
```suggestion
              href={`https://www.sanity.io/manage/project/${projectId}/members?invite=true`}
```

File: packages/sanity/src/core/i18n/bundles/studio.ts (hunk 3 above)
Comment (joneidejohnsen) [reply packages/sanity/src/core/i18n/bundles/studio.ts]: Check with someone from the Studio team or @runeb. They are located in https://github.com/sanity-io/locales, but I'm not sure about the process either.

**Review by ricokahler — APPROVED** [review]

**Pull request merged**